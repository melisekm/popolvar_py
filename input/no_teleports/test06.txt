5 30 100
HHHHHCHCCCHHHNHHHHNDHCCCCHCCCH
CCHCCCCCCHCCCCCNCHHHNCCCCCHCCH
CCCCCHPCCCHHCCHCCCCHNCCCHCCNCH
NCCCHHHHCHCCCHCCCCCHCCHCCCPNCH
NPNHCCHHCCCHHCHHHCCHCNNNNNNNCP