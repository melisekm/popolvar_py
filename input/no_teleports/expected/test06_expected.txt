0 0
0 1
1 1
2 1
3 1
4 1
5 1
6 1
7 1
8 1
9 1
10 1
11 1
12 1
13 1
14 1
14 2
15 2
16 2
17 2
18 2
18 1
19 1
19 0
20 0
21 0
21 1
22 1
23 1
24 1
25 1
26 1
27 1
28 1
28 2
28 3
28 4
29 4
28 4
28 3
28 2
28 1
27 1
26 1
26 2
26 3
25 3
24 3
23 3
22 3
21 3
20 3
19 3
18 3
17 3
16 3
15 3
14 3
13 3
12 3
11 3
10 3
9 3
8 3
8 2
7 2
6 2
5 2
4 2
3 2
3 3
2 3
1 3
1 4